torchrl/envs/transforms/transforms.py

```diff
@@ -2708,6 +2738,87 @@ def __init__(
         super().__init__(torch.double, torch.float, in_keys, in_keys_inv)
 
 
+class DeviceCastTransform(Transform):
+    """Casts the env device.
```

Comment:
Is this really casting the env device?

Isn't it transforming the device of the data?